node/openrpc.go

```diff
@@ -208,25 +210,59 @@ func newOpenRPCDocument() *go_openrpc_reflect.Document {
 }
 
 /*
-The following struct RPCSubscription and RPCSubscription.Unsubscribe method
+The following structs *RPCSubscription and *RPCSubscription.Unsubscribe method
```

Comment:
@meowsbits I want to play a bit in order to find a better way of merging those 3 duplicated implementations where only the **subscriptionName** type in `subscribe` changes.

Let me know if you have an idea.

Reply:
Have you experimented at all with getting rid of or modifying the [general "skip PubSub" case](https://github.com/etclabscore/core-geth/blob/735fd8804dae5f612b7bddea3b441767784caa04/node/openrpc.go#L143) to be more permissive (even if only allowing `debug` and `trace` receivers through)?

Reply:
No, I haven't. I will give it a try. Thanks

Reply:
I have tried it here, which adds the `trace_filter` nicely, although it can be used in the way documented, which leads to issues. What do you think? Did you mean something else @meowsbits?

![image](https://user-images.githubusercontent.com/802700/115776128-72322480-a3bc-11eb-9bc5-fd8d2bfdcc50.png)

Reply:
X-linking: https://github.com/etclabscore/core-geth/pull/364#pullrequestreview-643261538